### PR TITLE
Apply V3 Design / Text List (issue/2153)

### PR DESCRIPTION
**Pull request by stephanschroeter**
### Pull Request Checklist

- [x] I have read and accept the [Contributor License Agreement](https://github.com/porscheofficial/oss-docs/blob/67c5b24838a293ce7a964884e6005eb71f2b8579/CONTRIBUTOR_LICENSE_AGREEMENT.md)

#### References

- Preview: https://designsystem.porsche.com/issue/2153/components/text-list

#### Scope

Short description of the pull request scope

### CODE REVIEW

File: packages/components-js/tests/vrt/puppeteer/specs/component-scaling.vrt.ts [hunk 1]
```diff
@@ -35,6 +35,7 @@ const components = [
   'tag-dismissible',
   'text-field-wrapper',
   'textarea-wrapper',
+  'text-list',
```
Comment (stephanschroeter) [inline packages/components-js/tests/vrt/puppeteer/specs/component-scaling.vrt.ts:38]: add vrt test for multiline

File: packages/components/CHANGELOG.md [hunk 2]
```diff
@@ -415,8 +415,8 @@ to the new values since those ones will be removed with next major version.
 - `Spinner`, `Icon`, `Link Pure`, `Button Pure`, `Link`, `Link Social`, `Button`, `Checkbox Wrapper`,
   `Radio Button Wrapper`, `Popover`, `Modal`, `Select Wrapper`, `Tag`, `Tag Dismissible`, `Textarea Wrapper`,
   `Inline Notification`, `Banner`, `Toast`, `Grid`, `Flex`, `Pagination`, `Scroller`, `Accordion`, `Text`,
-  `Text Field Wrapper`, `Content Wrapper`, `Segmented Control`, `Tabs`, `Tabs Bar`, `Headline` and `Fieldset Wrapper`
```
Comment (stephanschroeter) [inline packages/components/CHANGELOG.md:418]: Move to unreleased

File: packages/components/src/components/text-list/text-list/__snapshots__/text-list-utils.spec.ts.snap [hunk 3]
```diff
@@ -0,0 +1,9 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`isListTypeOrdered() should return correct boolean for list type: ordered 1`] = `true`;
```
Comment (stephanschroeter) [inline packages/components/src/components/text-list/text-list/__snapshots__/text-list-utils.spec.ts.snap:3]: add expected in loop

**Review by denyo — APPROVED** [review]

File: packages/components/CHANGELOG.md [hunk 4]
```diff
@@ -29,7 +29,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 #### Changed
 
-- `Stepper Horizontal` matches new design language
```
Comment (denyo) [inline packages/components/CHANGELOG.md:32]: Wrong line 🙈

Comment (stephanschroeter) [reply packages/components/CHANGELOG.md]: I shouldn't work that long :P

**Pull request merged**